docs/tasks/federation/Policy-Engine-Service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: opa
+  namespace: federation-system
+spec:
+  selector:
+    app: opa
+  ports:
+  - name: http
+    protocol: TCP
+    port: 8181
+    targetPort: 8181
+  type: LoadBalancer
```

Comment:
It doesn't have to be exposed externally. I'd used this in the past to query the policy engine directly. Will remove.